Documentation/Signoffs/SpeedometerSystem.md

```diff
@@ -1,38 +1,34 @@
 SPEEDOMETER SUBSYSTEM
 --------------------
 ### Functions of the Subsystem
-The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on 4-digit a 7-segment display.
+The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on a LCD screen.
 ### Specifications & Constraints
--	Microcontroller must be compatible with MATLAB
--	Must be able to display the speed at which the user is going in the simulator
--	Shall connect to the PC running the simulation
--	Shall display the speed in real-time
--	Shall be powered by the PC	
-### Schematic
-##### 1. Pinouts
-
-![image](https://user-images.githubusercontent.com/117474540/202864081-4325652c-c3de-4a34-bf38-b649f1cd59cb.png)
+-	Microcontroller must be able to read data from MATLAB
+-	Shall connect to the PC running the simulation to obtain simulator data
```

Comment:
It should connect to only the PC. The Arduino attached to the LCD will not add more strain onto the PC. It will simply pull data from it.

Reply:
That disagrees with previous signoffs. Further, you have no basis for that claim.

Reply:
In our conceptual design this microcontroller was not connected to the master microcontroller. The microcontroller network is only used and involved with external and OEM sensors that send data to the simulation. This microcontroller that pulls data from the sim might add strain to the PC, but it will not add strain to the microcontroller network sending data into the simulation.

Reply:
Exactly. The PC was the concern according to prior signoffs.